src/transformStateWithClones.js

```diff
@@ -7,7 +7,39 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const result = [];
+  const clone = { ...state };
```

Comment:
it's better to specify what clone is that, `stateClone ` for example

Reply:
![image](https://user-images.githubusercontent.com/122989585/220053454-7cda40bb-1504-4b13-9709-50bde73b877c.png)